src/scheduler_pb2_grpc.py

```diff
@@ -0,0 +1,165 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
Aren't this auto generated? Can we remove all pb2 files and to README on how to generate it? Or better script it?

Reply:
Yeah good call. Script already exists. User would just need to ensure gRPC compiler is installed. I'll update README.